code/game/machinery/computer/arcade.dm

```diff
@@ -1027,3 +1027,278 @@
+	switch(unique_candidate)
+		if(UNIQUE_STEVE) // Steve is special
+			candidate_name = "Steve"
+			candidate_gender = MALE
+			age = "30"
+			cand_species = /datum/species/human
+			planet_of_origin = "Unknown"
+			job_requested = "Central Command Intern"
+			employment_records = "Experience in pressing buttons"
+		if(UNIQUE_MIME) // Only hire mimes that don't fill their employment application
+			candidate_name = "..."
+			candidate_gender = "..."
+			age = "..."
+			planet_of_origin = "..."
+			job_requested = "Mime"
+			employment_records = "..."
+		if(UNIQUE_CEO_CHILD) // Hes the son of the CEO, what do you expect?
+			candidate_name = "John Nanotrasen Junior"
+			candidate_gender = MALE
+			age = "12"
+			cand_species = /datum/species/human
+			planet_of_origin = "Unknown"
+			job_requested = "Captain"
+			employment_records = "Whatever"
+		if(UNIQUE_VIGILANTE) // For some reason vigilantes do get inside NT stations, let them slip in
+			candidate_name = "Owlman"
+			candidate_gender = MALE
+			age = "38"
+			cand_species = /datum/species/human
+			planet_of_origin = "Unknown"
+			job_requested = "Assistant"
+			employment_records = "Experience in hunting criminals"
+
+/obj/machinery/computer/arcade/recruiter/proc/win()
+	game_status = RECRUITER_STATUS_START
+	atom_say("Congratulations recruiter, the company is going to have a productive shift thanks to you.")
+	playsound(loc, 'sound/arcade/recruiter_win.ogg', 40)
+	prizevend(50)
+
+/obj/machinery/computer/arcade/recruiter/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "NTRecruiter", name, 400, 480)
+		ui.open()
+
+/obj/machinery/computer/arcade/recruiter/ui_data(mob/user)
+	var/list/data = list(
+	"gamestatus" = game_status,
+
+	"cand_name" = candidate_name,
+	"cand_gender" = capitalize(candidate_gender),
+	"cand_age" = age,
+	"cand_species" = initial(cand_species.name),
+	"cand_planet" = planet_of_origin,
+	"cand_job" = job_requested,
+	"cand_records" = employment_records,
+
+	"cand_curriculum" = curriculums,
+
+	"reason" = reason
+	)
+	return data
+
+/obj/machinery/computer/arcade/recruiter/ui_act(action, list/params, datum/tgui/ui)
+	if(..())
+		return
+
+	var/mob/user = ui.user
+	add_fingerprint(user)
+	. = TRUE
+
+	switch(action)
+		if("hire")
+			playsound(user, 'sound/items/handling/standard_stamp.ogg', 50, TRUE)
+			if(!good_candidate)
+				game_status = RECRUITER_STATUS_GAMEOVER
+				playsound(loc, 'sound/misc/compiler-failure.ogg', 2, TRUE)
+				reason = "You ended up hiring incompetent candidates and now the company is wasting lots of resources to fix what you caused..."
+			else if(curriculums >= 5)
+				win()
+			else
+				curriculums++
+				good_candidate = TRUE
+				if(prob(PROB_UNIQUE_CANDIDATE))
+					unique_candidate()
+				else
+					generate_candidate()
```

Comment:
```suggestion
			if(curriculums >= 5)
				win()
				return
			curriculums++
			good_candidate = TRUE
			if(prob(PROB_UNIQUE_CANDIDATE))
				unique_candidate()
			else
				generate_candidate()
```
I think this'd be a bit cleaner with making this an early return

Reply:
added in next commit